src/LoginView.js

```diff
@@ -28,10 +28,12 @@ export default class LoginView extends PureComponent {
   }
   _backHandler() {
     const { uri } = this.state;
-    if (uri == adService.getPasswordResetURI()) {
-      this._handleFlowResultAsync({ requestType: "cancelled" }, uri);
+    if (this.webView) {
```

Comment:
Any rationale?

Reply:
Also add a comment describing why this is necessary.

Reply:
Error comes from this.webView.goBack is a nullpointer. Happens if backPress is pressed before the webView ref has been established. Just add check if the webView is has been initated to avoid it. Return true anyways to avoid exiting the app. Check backHandler doc for reference.

Reply:
Kool. Add a comment for future reference?

Reply:
jupp